src/routes/sig.py

```
@@ -100,6 +100,23 @@ async def handover_sig(id: int, session: SessionDep, request: Request, body: Bod
     return
 
 
+@sig_router.post('/executive/sig/{id}/handover', status_code=204)
+async def executive_handover_sig(id: int, session: SessionDep, request: Request, body: BodyHandoverSIG) -> None:
+    current_user = get_user(request)
+    if current_user.role < get_user_role_level('executive'): raise HTTPException(403, detail="임원진만 시그/피그장을 양도할 수 있습니다")
```

Ruff (0.13.3)

Multiple statements on one line (colon)

(E701)